addons/point_of_sale/tests/test_frontend.py

```diff
@@ -13,6 +13,10 @@
 
 class TestPointOfSaleHttpCommon(AccountTestInvoicingHttpCommon):
 
+    @classmethod
+    def _get_main_company(cls):
+        return cls.company_data['company']
+
```

Comment:
I don't the need for this refactoring. Remove this method if not reused.

Reply:
it's used